core/templates/dev/head/components/ProfileLinkImageDirective.js

```diff
@@ -32,18 +32,21 @@ oppia.directive('profileLinkImage', [function() {
         return GLOBALS.SYSTEM_USERNAMES.indexOf(username) == -1;
       };
 
-      $scope.profileImageUrl = (
-        '/preferenceshandler/profile_picture_by_username/' + $scope.username());
-      $scope.profilePicture = DEFAULT_PROFILE_IMAGE_PATH;
+      if ($scope.username() !== 'Community Owned') {
```

Comment:
I suggest creating a profile-picture-sized image for the globe, and using that instead.

Another alternative is to create a community-editable-indicator directive that displays a reusable "community-editable" badge that can be applied anywhere, but you might find that this introduces too much custom logic when deciding how to display the avatars on the summary tiles. Up to you, though!